src/common/Card/Reminder/Reminder.tsx

```diff
@@ -20,14 +25,16 @@ export const Reminder: React.FC<ReminderProps> = ({ startTime, duration }) => {
   }, []);
 
   /* Displays the reminder status if start time is less than 15 minutes away */
-  if (startTime.getTime() - currentTime.getTime() < 15 * 60000) {
-    let message = "Starts in " + mins.toString() + " minutes";
+  if (startTime.getTime() - currentTime.getTime() < FIFTEEN_MIN_MS) {
+    let message = `Starts in ${mins.toString()} minutes`;
     if (mins < 0) {
       if (duration + mins > 0) {
         message = "Ongoing";
-      } else {
+      } else if (minutesSinceEnded < ONE_DAY_MINUTES) {
         message = "Ended";
         return <UI.StyledExpiredReminder>{message}</UI.StyledExpiredReminder>;
+      } else {
+        return <></>;
       }
```

Comment:
Add comments to this block to make logic more obvious or simplify it. Using the same unit of time might help.